lab_2_keywords_cooccurrence/main.py

```diff
@@ -84,7 +167,19 @@ def calculate_cumulative_score_for_candidates(candidate_keyword_phrases: KeyPhra
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if (not isinstance(candidate_keyword_phrases, list) or candidate_keyword_phrases == []
+            or not isinstance(word_scores, dict) or word_scores == {}):
```

Comment:
```suggestion
            or not isinstance(word_scores, dict) or not word_scores):
```

Reply:
Ant every time when you use `smt == {}` or `smt == []` use `NOT smth`

Reply:
understood, done